### ISSUE TITLE
Headings and subheadings should use proper HTML elements

### ISSUE THREAD

**Issue by Andrew-Dame** [international-child-art-foundation/arts-olympiad#19]
For any headings and subheadings, the HTML elements `h1`...`h6` should be used to provide better context to screen readers and SEO. 

As an example, the "purpose" page should follow a structure like so:
```
<h1>Purpose</h1>
...

<h2>Engaging Creative Kids in the Olympics</h2>

...

<h2>The Arts Olympiad</h2>

...

<h2>ICAF & the Olympics </h2>

...
```

More information on best practices: 

https://www.w3.org/WAI/tutorials/page-structure/headings
